Makefile

```diff
@@ -1,30 +1,39 @@
 SHELL = /bin/sh
 
+EXECUTABLES = docker bats jq
+CHECK := $(foreach executable,$(EXECUTABLES),\
+	$(if $(shell which $(executable)),"",$(error "No executable found for $(executable).")))
+
 DOCKER ?= $(shell which docker)
 DOCKER_REPOSITORY := graze/composer
 
 BATS ?= $(shell which bats)
 
-EXECUTABLES = docker bats jq
-CHECK := $(foreach executable,$(EXECUTABLES),\
-	$(if $(shell which $(executable)),"",$(error "No executable found for $(executable).")))
-
 .PHONY: images test clean help
 
 .SILENT: help
 
 default: help
 
 images-quick: ## Build the image 🚀(quicker).
-	${DOCKER} build --tag ${DOCKER_REPOSITORY}:latest --tag ${DOCKER_REPOSITORY}:php-7.0 ./php-7.0
-	${DOCKER} build --tag ${DOCKER_REPOSITORY}:php-5.6 ./php-5.6
+	${DOCKER} build --tag ${DOCKER_REPOSITORY}:latest \
+		--tag ${DOCKER_REPOSITORY}:php-7 \
+		--tag ${DOCKER_REPOSITORY}:php-7.1 ./php-7.1
+	${DOCKER} build --tag ${DOCKER_REPOSITORY}:php-7.0 ./php-7.0
+	${DOCKER} build --tag ${DOCKER_REPOSITORY}:php-5.6 \
+		--tag ${DOCKER_REPOSITORY}:php-5 ./php-5.6
 
 images: ## Build the image 🚀.
-	${DOCKER} build --no-cache --pull --tag ${DOCKER_REPOSITORY}:latest --tag ${DOCKER_REPOSITORY}:php-7.0 ./php-7.0
-	${DOCKER} build --no-cache --pull --tag ${DOCKER_REPOSITORY}:php-5.6 ./php-5.6
+	${DOCKER} build --no-cache --pull --tag ${DOCKER_REPOSITORY}:latest \
+		--tag ${DOCKER_REPOSITORY}:php-7.1 \
+		--tag ${DOCKER_REPOSITORY}:php-7 ./php-7.1
+	${DOCKER} build --no-cache --pull --tag ${DOCKER_REPOSITORY}:php-7.0 ./php-7.0
+	${DOCKER} build --no-cache --pull --tag ${DOCKER_REPOSITORY}:php-5.6 \
+		 --tag ${DOCKER_REPOSITORY}:php-5 ./php-5.6
```

Comment:
To avoid the `images`/`images-quick` repetition (which will increase as we add more PHP versions) I suggest refactoring them. Something like this perhaps:

```bash
images-quick: ## Build the image 🚀(quicker).
    DOCKER_OPTIONS="" make images-base

images: ## Build the image 🚀.
    DOCKER_OPTIONS=" --no-cache --pull" make images-base

images-base:
   ${DOCKER} build $(DOCKER_OPTIONS) --tag ${DOCKER_REPOSITORY}:latest \
        --tag ${DOCKER_REPOSITORY}:php-7.1 \
        --tag ${DOCKER_REPOSITORY}:php-7 ./php-7.1
   ${DOCKER} build $(DOCKER_OPTIONS) --tag ${DOCKER_REPOSITORY}:php-7.0 ./php-7.0
   ${DOCKER} build $(DOCKER_OPTIONS) --tag ${DOCKER_REPOSITORY}:php-5.6 \
         --tag ${DOCKER_REPOSITORY}:php-5 ./php-5.6
```